app/js/components/users/Show.react.js

```diff
@@ -106,19 +106,20 @@ class Show extends React.Component {
           <div className="ui grid">
             <div className="sixteen wide tablet twelve wide computer column">
                      <div className="ui divider"></div>
-              <h2>Top Questions</h2>
+              <h2>My Questions</h2>
             {this.state.userQuestions ? this.state.userQuestions.splice(0,2).map(this.createUserQuestionsDiv) : `<div />`}
           </div>
           </div>
 
           <div className="ui grid">
             <div className="sixteen wide tablet twelve wide computer column">
                 <div className="ui divider"></div>
-                <h2>Top Answers</h2>
+                <h2>Answers by me</h2>
                 {[1,3].map(this.userAnswers)}
             </div>
           </div>
 
+          {<TagModal  options={{modalId, closable: true}} />}
```

Comment:
The modal you mean? We'd have to discuss about that. Weigh advantages and disadvantages